lib/zendesk_apps_support/package.rb

```diff
@@ -173,25 +279,8 @@ def file_path(path)
       File.join(root, path)
```

Comment:
Looks like `file_path` is called from [`Validations::Banner`](https://github.com/zendesk/zendesk_apps_support/blob/olaf/multiple_apps/lib/zendesk_apps_support/validations/banner.rb#L11), so it can't be private.